lib/metadata/index.js

```diff
@@ -125,6 +125,11 @@ Metadata.prototype.buildTokens = function (hosts) {
   this.primaryReplicas = primaryReplicas;
   //All the tokens in ring order
   this.ring = allSorted;
+  //Compute string versions as it's potentially expensive and frequently reused later
+  this.ringTokensAsStrings = [];
+  for (var i = 0; i < allSorted; i++) {
```

Comment:
Oh that's embarrassing. 

I've had a look through and can't see any unit tests any unit tests around this function which I could have modified to pick this error up. Am I missing it or is it just covered in the ci/integration tests (which I haven't managed to get running locally)?